releasenotes/notes/add_known_issue_rcm_gevent_releasenote-1658a344a9aaf1b5.yaml

```
@@ -0,0 +1,5 @@
+---
+issues:
+  - |
+    ASM: There is a known issue with remote configuration management (RCM) when paired with gevent which can cause workers to hang. 
+    If you are using gevent and ASM (``DD_APPSEC_ENABLED=true``), we recommend disabling RCM with ``DD_REMOTE_CONFIGURATION_ENABLED=false``.
```

Would users even know what RCM is? Should we call out one click activation instead?
```suggestion
    gevent: There is a known issue with remote configuration management (RCM) when paired with gevent which can cause workers to hang. 
    If you are using gevent, we recommend disabling RCM with ``DD_REMOTE_CONFIGURATION_ENABLED=false``. Note, this will disable one click activate.
```

---

👍🏻 to the note about removing one-click activation if you do this. Good call out.

I wonder if it should be `remote config: ... ` instead of `gevent:` or `ASM:` ?

---

I don't know, would users know what one click activation is? Neither is in our corp docs.

---

It is defined in our release notes for v1.6.0 🤷🏻

---

talked with Brett, will add the note, and change scope to `remote config`